modules/spo-uninstall-console.adoc

```diff
@@ -10,7 +10,7 @@ To remove the Security Profiles Operator, you must first delete the `seccomp` an
 
 .Prerequisites
 
-* Access to an {product-title} cluster that uses an account with `cluster-admin` permissions.
+* Have access to the web console as a user with `cluster-admin` privileges.
```

Comment:
I'd rewrite this as a full sentence to be consistent with the full sentence in the second bullet.
```suggestion
* You have access to the web console as a user with `cluster-admin` privileges.
```